pt/lessons/libraries/benchee.md

```diff
@@ -111,27 +113,31 @@ Benchee.run(%{"example function" => fn -> "hi!" end}, [
     comparison: true,
     unit_scaling: :best
   ]
-])
+)
 ```
 
 As opções disponíveis são as seguintes (também documentadas em [hexdocs](https://hexdocs.pm/benchee/Benchee.Configuration.html#init/1)).
 
 * **warmup** - o tempo em segundos para o qual um cenário de _benchmarking_ deve ser executado sem tempos de medição antes do início das medidas reais. Isso simula um sistema de funcionamento "quente". Padrão é 2.
 * **time** - o tempo em segundos por quanto tempo cada cenário de _benchmarking_ individual deve ser executado e medido. Padrão é 5.
-* **inputs** - um mapa com _strings_ que representam o nome de entrada como as chaves e a entrada real como valores. Padrão é `nil`. Vamos abordá-lo em detalhes na próxima seção.
+* **memory_time** - o tempo em segundos de quanto tempo o consumo de memória deve ser medido para cada cenário do benchmarking. Veremos isso mais tarde. O padrão é 0.
+* **inputs** - um mapa com strings que representam o nome da entrada como as chaves e a entrada real como os valores. Também pode ser uma lista de tuplas no form `{input_name, actual_value}`. O padrão é `nil` (sem entradas). Vamos cobrir isso em detalhes na próxima seção.
```

Comment:
"... uma lista de tuplas no _form_ ..." I believe maybe a better translation here would be "uma lista de tuplas no _formato_ ..."?

Reply:
yes I sent this way use a `form` word because I couldn't figure out the best word in Portuguese to use here, thanks I will send a quick fix